README.md

```diff
@@ -10,10 +10,11 @@ pip install jupysql-plugin
 ### Development install
 
 ```sh
- conda create --name jupysql-plugin python=3.11 --channel conda-forge --yes
+conda create --name jupysql-plugin python=3.11 --channel conda-forge --yes
 conda activate jupysql-plugin
 conda install nodejs=16 --channel conda-forge --yes
 pip install -r requirements.txt
+pip install -r requirements.dev.txt
```

Comment:
i see this new file has been added and `pkgmt` has been added, but I don't see why it's needed.

Reply:
ok, so I was missing that context because I saw no mentions to `pkgmt lint`. then, just add documentation that `pkgmt lint` is used for linting